react/features/chat/components/native/ChatInputBar.tsx

```diff
@@ -66,11 +74,18 @@ class ChatInputBar extends Component<IProps, IState> {
      * @inheritdoc
      */
     render() {
+        let inputBarStyles;
+
+        if (this.props.aspectRatio === ASPECT_RATIO_WIDE) {
+            inputBarStyles = styles.inputBarWide;
+        } else {
+            inputBarStyles = styles.inputBarNarrow;
+        }
+
         return (
-            <SafeAreaView
```

Comment:
Why are we dropping this? Does it no longer help?

Reply:
No, there is no need for it anymore.

Reply:
But we seem to position things using hardcoded offsets now right? Isn't that worse.

Reply:
With or without safeareaview, we need to set a fixed height for this component. I can add back this safe area view and remove some of the height value.

Reply:
Ah nevermind, I see it was the wrapper of ChatInout now. Approved!